Module3/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DeliveryService.Data;
+
+namespace DeliveryService
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var data = new StoreContext();
+            Console.WriteLine("-----Task1-----");
+            Task1(data);
+            Console.WriteLine("-----Task2-----");
+            Task2(data);
+            Console.WriteLine("-----Task3-----");
+            Task3(data);
+            Console.WriteLine("-----Task4-----");
+            Task4(data);
+            Console.WriteLine("-----Task5-----");
+            Task5(data);
+        }
+
+        static void Task1(StoreContext data)
+        {
+            var products = data.Products.OrderBy(p => p.Name).ToList();
+            foreach(var product in products)
+            {
+                Console.WriteLine($"{product.Name}");
+            }
+        }
+
+        static void Task2(StoreContext data)
+        {
+            var products = data.Products.Join(data.Users,
```

Comment:
lets add version where u should  Seller info via Product
use Product-Seller relation